app/mailers/user_mailer.rb

```diff
@@ -1,6 +1,11 @@
 class UserMailer < ApplicationMailer
   default from: "Epic Logger <admin@epiclogger.com>"
 
+  def email_confirmation(token)
```

Comment:
Also here add a `return if user.confirmed?` by making a `def confirmed?` method in the user model to check if the user is confirmed. That will stop the email sending if the user was confirmed.